application/views/contesting/index.php

```diff
@@ -379,7 +379,7 @@
                                         </div>
                                 </div>
                             <button type="reset" class="btn btn-sm btn-warning" onclick="reset_log_fields()">Reset</button>
-                            <button type="button" class="btn btn-sm btn-primary" onclick="logQso();"><i class="fas fa-save"></i> Log QSO</button>
```

Comment:
"Save QSO" was used elsewhere so I'm aiming to standardise on that.

Reply:
I don't think that saving a QSO is the same as logging a QSO... here we are in Contest-Context creating new log-entries, with "save" you are in edtiting context...

Reply:
Good point. Let me go back and review the current usage again.

Reply:
I feel that "Log QSO" is more correct, but you are also saving the QSO in the database. I named it "Log QSO" in the Contest part.

Reply:
@magicbug, would you like:

- "Save QSO" all around. I'll keep this commit as-is.
- "Log QSO" all around. I'll change the buttons in the Live QSO and Post QSO pages to match the Contest Logging page and I'll undo this part of my commit.